addons/html_editor/static/src/fields/html_field.js

```diff
@@ -31,6 +31,10 @@ export class HtmlField extends Component {
         isCollaborative: { type: Boolean, optional: true },
         cssReadonlyAssetId: { type: String, optional: true },
         sandboxedPreview: { type: Boolean, optional: true },
+        editorConfig: { type: Object, optional: true },
+    };
+    static defaultProps = {
+        editorConfig: {},
```

Comment:
be careful with static props like this: the editorConfig object here is shared between all instance of the field that do not have a editorConfig prop. You should remove it, in your case, it's not needed. you can do `{...editorConfig}` even if it is undefined anyway